.gitignore

```diff
@@ -2,10 +2,7 @@
 .idea/
 .vscode/
 deps/
-examples/librimix/MiniLibriMix/
-examples/librimix/MiniLibriMix.zip
-examples/librimix/librimix_nb/
-examples/librimix/librimix/
+examples/*/env/
```

Comment:
I'm not sure about the `env` name - it collides with the `virtualenv` directory naming convention, suggesting that we are having a separate Python env for each recipe. How about an old-fashioned `data` instead?

Reply:
Since the main purpose of  lhotse is dealing with data, I think the name `data` would be okay.

Reply:
Great! Could you rename? I recall it's not the only place where this name is used.